### PR TITLE
Minor document changes

### PR DESCRIPTION
**Pull request by dhairyya**
Allows are not automatically extended by us
Allow and blocks are both supported by spoof sender is no longer needed after document structuring
Notes for blocked spoof senders updated

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/security/office-365-security/allow-block-email-spoof.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ For detailed syntax and parameter information, see [New-TenantAllowBlockListItem
 
 Allow senders (or domains) on the **Submissions** page in Microsoft 365 Defender.
 
-Note that admins can't add allows directly to the Tenant Allow/Block List. Instead, you use the admin submission process to submit the message that were blocked so the corresponding URL, file, and/or senders will be added to the Tenant Allow/Block List. If a block of the file, URL, or sender has not happened, then the allow will not be created. In most cases where the message was determined to be a false positive that was incorrectly blocked, the allows are kept for as long as needed to give the system time to allow them naturally.
+Note that admins can't add allows directly to the Tenant Allow/Block List. Instead, you use the admin submission process to submit the message that were blocked so the corresponding URL, file, and/or senders will be added to the Tenant Allow/Block List. If a block of the file, URL, or sender has not happened, then the allow will not be created. In most cases where the message was determined to be a false positive that was incorrectly blocked, the allows are removed on the date of the expiry as set by you while creating it.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/allow-block-email-spoof.md]:
```suggestion
Note that admins can't add allows directly to the Tenant Allow/Block List. Instead, you use the admin submission process to submit the message that was blocked so the corresponding URL, file, and/or senders will be added to the Tenant Allow/Block List. If a block of the file, URL, or sender has not happened, then the allow will not be created. In most cases where the message was determined to be a false positive that was incorrectly blocked, the allows are removed on the date of the expiry as set by you while creating it.
```

File: microsoft-365/security/office-365-security/allow-block-email-spoof.md [hunk 2]
```diff
@@ -92,7 +92,9 @@ Note that admins can't add allows directly to the Tenant Allow/Block List. Inste
 
 6. From the **Remove after** drop-down list, specify how long you want the allow option to work.
 
-7. When you're finished, click the **Submit** button.
+7. Add why you are adding allow using the **Optional Note**. 
+
+8. When you're finished, click the **Submit** button.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/allow-block-email-spoof.md]:
```suggestion
8. When you're finished, select the **Submit** button.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**